core/src/main/java/com/novoda/noplayer/exoplayer/ExoPlayerCreator.java

```diff
@@ -18,8 +19,8 @@
         this.trackSelector = trackSelector;
     }
 
-    public SimpleExoPlayer create() {
-        DefaultRenderersFactory renderersFactory = new DefaultRenderersFactory(context);
+    public SimpleExoPlayer create(DrmSessionCreator drmSessionCreator) {
```

Comment:
It seems like it's injected at a later stage than this class is initialised, upstream